VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-chat/vho-chat.component.spec.ts

```diff
@@ -75,7 +75,7 @@ describe('VhoChatComponent', () => {
             id: Guid.create().toString(),
             from: username,
             message: 'test message',
-            timestamp: new Date()
+            timestamp: new Date(),
```

Comment:
There are no changes here. Could you revert lines that have trailing commas please.

Reply:
done